Dockerfile

```diff
@@ -10,7 +10,11 @@ ENV PATH="/flutter/bin:${PATH}"
 
 # Running `flutter config --no-analytics` downloads the Dart SDK and
 # disables analytics tracking – which we always want
+# `1a1bbacf0d` maps to alpha release 0.0.8 @ June 13, 2017
+#    We hard wire the SHA to ensure Docker rebuilds
```

Comment:
I think so.

The case I'm worried about: we pin to some  tag and then someone decides to update it – we'll have to fall back to this logic.

Having it explicit makes it very clear where we're pointed – there's no notion of "0.0.8" as of June 25